classes/Profile.php

```diff
@@ -52,6 +52,16 @@ class ProfileCore extends ObjectModel
 
     protected static $_cache_accesses = [];
 
+    /**
+     * {@inheritdoc}
+     */
+    public function __construct($id = null, $idLang = null, $idShop = null, $translator = null)
+    {
+        parent::__construct($id, $idLang, $idShop, $translator);
+
+        $this->image_dir = _PS_PROFILE_IMG_DIR_;
```

Comment:
Why adding this property? :thinking:

Reply:
For keeping the same coherence on each legacy class which has some images : 
![image](https://user-images.githubusercontent.com/1533248/93354079-eca23100-f83c-11ea-94c8-d4cf53883383.png)

Reply:
I love legacy parts...